### PR TITLE
#83 corrections of installation instructions

### PR DESCRIPTION
**Pull request by pbroman**
Corrections of some inconsistencies in installations instructions and makefile.
I hope I got it all right, please verify.

### CODE REVIEW

File: INSTALL.md [hunk 1]
```diff
@@ -40,7 +40,7 @@ make apt-get
 Install current Git revisions of hocr-tools and ocropus:
 
 ```
-make dist/vendor
+make vendor
```
Comment (stweil) [inline INSTALL.md:43]: Looks good.

File: INSTALL.md [hunk 2]
```diff
@@ -73,18 +73,18 @@ This will recreate out-of-date files in `./dist`, create a folder
 `$APACHE_BASEURL` in `$APACHE_DIR` and copy all the files from `./dist` to
 `$APACHE_DIR/$APACHE_BASEURL` using `sudo` with user `$APACHE_USER`.
 
-Deployment can be customized with three environment variables, the default is:
+Deployment can be customized with four environment variables, the default is:
```
Comment (stweil) [inline INSTALL.md:76]: Looks good.

File: INSTALL.md [hunk 3]
```diff
@@ -73,18 +73,18 @@ This will recreate out-of-date files in `./dist`, create a folder
...
 
 ```
-make APACHE_USER=www-default APACHE_DIR=/var/www/html APACHE_BASEURL=ocr-gt-tools deploy
+make APACHE_USER=www-data APACHE_GROUP=www-data APACHE_DIR=/var/www/html APACHE_BASEURL=ocr-gt deploy
```
Comment (stweil) [inline INSTALL.md:79]: Looks good. @kba, @zuphilip, `ocr-gt` is the current default for the base URL. Is this what we want, or should it be fixed to be `ocr-gt-tools`?

File: Makefile [hunk 4]
```diff
@@ -15,6 +15,8 @@ include \
 MKDIR = mkdir -p
 RM    = rm -rf
 CP    = cp -r
+GIT_CLONE = git clone --depth 1
```
Comment (stweil) [inline Makefile:18]: Looks good. This is definitely needed.

**Review by stweil — APPROVED** [review]
I think this PR can be merged (unless we want to fix the base URL to be `ocr-gt-tools`.

**Review by kba — APPROVED** [review]

**Comment by stweil** [comment]
I added a small typo fix for INSTALL.md. Thank you, @pbroman, for your pull request, and sorry for the long time it took to get it merged.

**Pull request merged**